ntoskrnl/mm/balance.c

```diff
@@ -39,6 +38,23 @@ static LONG PageOutThreadActive;
 
 /* FUNCTIONS ****************************************************************/
 
+static inline VOID UpdateTotalCommittedPages(VOID)
```

Comment:
That inline function should be moved into the mm.h (and the "static" be removed).

Reply:
ok